<launch>
    <arg name="visual_scanner" default="/zed2/scan" />
    <node name="navServer" pkg="actions" type="navServer.py" output="screen">
        <param name="visual_scanner" value="$(arg visual_scanner)" />
    </node>
    <node name="navClient" pkg="actions" type="navClient.py" output="screen" />
</launch>